tests/test_z6_store_contract.py

```
@@ -0,0 +1,139 @@
+    assert children == {"zarr.json", "c"}
+
+
+@pytest.mark.asyncio
+async def test_root_level_chunk_key_no_silent_clobber() -> None:
+    store = await populated_store()
+
+    try:
+        await store.set("c/0/0", buf(b"ROOT-CHUNK"))
+    except Exception:
+        # Rejecting root-level chunks explicitly is an acceptable contract.
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)